Singer.API/ClientApp/src/app/modules/core/models/singer-colors.ts

```diff
@@ -0,0 +1,154 @@
+/*
```

Comment:
Because I can't acces SCSS variables from inside TS directly without using hacks, the point of this class is to have the colors available in TS too. In our current use case the detailed user cards have an HTML color atribute that can be used to set the background color of the Little initials circle at the top. These TS variables and functions help us do this in a handy way providing customization and randomizing.

Reply:
This is the alternative: https://stackoverflow.com/questions/40418804/access-sass-values-colors-from-variables-scss-in-typescript-angular2-ionic2

It's not that bad but i preffered the namespace and the use of enums because thay make picking color values trough a function based on numbers like an index or a random number easy. If neccessary this can be changed too.

Reply:
I don't know of a better way to accomplish this right now, so perhaps we can leave it as is for now. @BerendWouters maybe you have another insight?

Reply:
I have an idea💡

Reply:
Nice 👍